/home/luc/hgwork/lino/docs/api/lino.modlib.reception.rst:14: WARNING: autodoc: failed to import module u'lino.modlib.reception.models'; the following exception was raised:
Traceback (most recent call last):
  File "/home/luc/pythonenvs/py27/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 326, in import_object
    __import__(self.modname)
  File "/home/luc/hgwork/lino/lino/modlib/reception/models.py", line 391, in <module>
    ExpectedGuestsStates = (GuestStates.invited,GuestStates.accepted)
AttributeError: type object 'GuestStates' has no attribute 'accepted'
